client/me/purchases/vat-info/style.scss

```diff
@@ -18,6 +18,14 @@
 .vat-info__sidebar-paragraph {
 	margin-bottom: 0;
 	font-size: 0.875rem;
+
+	.vat-info__open-help-center-support {
+		font-size: 0.875rem;
+		text-decoration: none;
+		&:hover, &:focus, &:active {
+			color: var( --color-link-dark );
+		}
+	}
```

Comment:
I don't see a good reason for this to be a `Button` from `@wordpress/components`. Can't it just be a `<a />`? That way we won't have to declare all these overrides and we can rely safely on the defaults.

Reply:
I chose the button for semantic reasons on this interaction. I can follow up with another PR and update this to an anchor tag to remove the overrides.

Reply:
Can you please clarify which semantical advantages of `button` were reasonable to justify this decision?

Reply:
It is more action-oriented (rather than for navigation) and comes with built-in accessibility support. If we prefer to avoid dealing with these CSS overrides, I can find a way to update the accessibility of the anchor tag to better convey that it's being used as a button.

Reply:
I'm aware of the theoretical differences between anchors and buttons. I was asking about this specific case - what makes the link less accessible here? It feels like the benefits of `button` are not very relevant here, other than to semantically signal that this is interactive behavior and not a navigation, which is of little practical value. If we really want to do this right, we might want to do it centrally, but TBH I'd prefer not having to deal with the CSS overrides in the first place. Ideally we should have none of them.

Reply:
Thanks for the feedback. I'm addressing this in another PR soon.